src/drivers/imx/interrupt.c

```diff
@@ -363,6 +363,20 @@ static const struct irq_cascade_tmpl dsp_irq[] = {
 	IRQSTR_CASCADE_TMPL_DECL(7)
 };
 
+int irqstr_get_sof_int(int irqstr_int)
+{
+	int line, irq;
+
+	/* Is it a valid interrupt? */
+	if (irqstr_int < 0 || irqstr_int >= 512)
+		return -EINVAL;
+
+	line = irqstr_int / 64;
```

Comment:
Ditto for 64.

Reply:
Added new define, also replacing throughout driver. Is that fine? Pushing now.